app/src/main/java/org/oppia/app/profile/AddProfileFragment.kt

```diff
@@ -0,0 +1,26 @@
+package org.oppia.app.profile
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.oppia.app.fragment.InjectableFragment
+import javax.inject.Inject
+
+/** Fragment that allows users to create new profiles */
```

Comment:
Nit: please end the KDoc in punctuation. Ditto for other files in this PR.

Reply:
+1